pages/calldata-decoder/index.page.tsx

```diff
@@ -242,6 +242,7 @@ export default function CalldataDecoder(): ReactElement {
         <span>Calldata</span>
       </label>
       <>
+        {/* todo */}
```

Comment:
todo what? can you more context: what is exactly left to be done?

Reply:
This is still draft and it's not finished